src/styles/main.scss

```diff
@@ -1,3 +1,242 @@
+$headerLinkBorderColor: #00acdc;
+$cardWidth: 200px;
+$distanceBetweenCards: 48px;
+
+@function mainContainerSize($width, $distanceBetween) {
+  @return ($width * 4) + ($distanceBetween * 3);
+}
+
+// #region normalize
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+// #endregion
+
+// #region header
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+
+  &__logoLink {
+    height: 40px;
+  }
+}
+// #endregion
+
+// #region list
+.list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  height: 60px;
+
+  &__items {
+    margin: 0 10px;
+  }
+
+  &__items:nth-child(1) {
+    margin-left: 0;
+  }
+
+  &__items:last-child {
+    margin-right: 0;
+  }
+
+  &__link {
+    position: relative;
+    padding: 22.5px 0;
+    display: flex;
+    text-decoration: none;
+    color: #000;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 14px;
+    height: 15px;
+    transition: 350ms;
+  }
+
+  &__link:hover {
+    color: $headerLinkBorderColor;
+  }
+}
+// #endregion
+
+// #region headerLinkActive
+.is-active {
+  color: $headerLinkBorderColor;
+
+  &::after {
+    position: absolute;
+    content: "";
+    height: 4px;
+    right: 0;
+    bottom: -15px;
+    left: 0;
+    background-color: $headerLinkBorderColor;
+    border-radius: 8px;
+  }
+}
+
+// #endregion
+
+// #region main
+.main {
+  display: flex;
+  justify-content: center;
+  padding-top: 50px;
+
+  &__container {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    width: mainContainerSize($cardWidth, $distanceBetweenCards);
+  }
+}
+// #endregion
+
+// #region card
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $cardWidth;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-right: $distanceBetweenCards;
+  margin-bottom: 50px;
```

Comment:
card can't have outer spacing, use mix class for this
https://www.youtube.com/watch?v=JOJ-mENrg_s

Reply:
tests don`t see gap